llvm/test/CodeGen/DirectX/rootsignature-validation.ll

```diff
@@ -0,0 +1,28 @@
+; RUN: opt -S -passes='dxil-post-optimization-validation' -mtriple=dxil-pc-shadermodel6.6-compute %s 
+; expected-no-diagnostics
```

Comment:
The `expected-no-diagnostics` syntax is for `clang -verify`. Might be clearer to just write it out in a sentence like "We have a valid root signature, this should compile successfully"